modules/rank-eval/src/main/java/org/elasticsearch/index/rankeval/QuerySpec.java

```diff
@@ -32,25 +36,33 @@
  * Defines a QA specification: All end user supplied query intents will be mapped to the search request specified in this search request
  * template and executed against the targetIndex given. Any filters that should be applied in the target system can be specified as well.
  *
- * The resulting document lists can then be compared against what was specified in the set of rated documents as part of a QAQuery.  
+ * The resulting document lists can then be compared against what was specified in the set of rated documents as part of a QAQuery.
  * */
 public class QuerySpec implements Writeable {
 
-    private int specId = 0;
+    private String specId;
```

Comment:
I changed this to a String because in the rest request examples I found things like "amsterdam_query" etc. I think its good to have something human readable here.